### PR TITLE
Added Mocha Unit Test

### PR DESCRIPTION
**Pull request by jaidetree**
Hey there so I found myself needed a library to pack & compress a JSON object for a URL and this does the trick. However, I wanted a simple test before implementing it so chose to setup a basic test to make sure the compressor and decompressor are doing their jobs.

Some build errors were experienced with the original library versions so I updated those and found it still works.

Let me know if anything is not up to snuff.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+let expect = require('expect'),
+    jsonurl = require('.'),
```
Comment (masotime) [inline test.js]: I had some difficulties running it with `require('.')`. Changing it to `require('./index')` seems to help. Or possibly move this file to a `/test` folder?

**Comment by masotime** [comment]
Hey @jayzawrotny, thanks for taking the time to write the tests! They look great, but I was having difficulty with running them locally.

Could you describe your dev environment? I see that you're using ES6, but I'm not sure how you run your test.

I've added some comments in the code which are related to the issues I'm seeing. 

Also, perhaps you can add something to `package.json` for a test script, e.g. `./node_modules/.bin/mocha --compilers js:babel-core/register` - from what I can tell, this only works with babel-core@^5. We'll need to add `babel-core` as a `devDependency` of course.

File: test.js (hunk 1 above)
Comment (masotime) [inline test.js]: Actually just `require('./')` works as well.

**Comment by jaidetree** [comment]
Ah, sorry for the trouble. I am running Node 5 which supports the ES6 features that are natively available https://nodejs.org/en/docs/es6/. I'll just replace the ES6 features with their ES5 implementations to keep things simple. As for running the moca tests just `./node_modules/.bin/mocha test.js` will work on Node 5 and above but when the ES5 versions are done older versions of node should be supported as well.

**Comment by masotime** [comment]
Heh ok it's so minor I'll make the change.

**Comment by masotime** [comment]
Published as v1.0.4

**Comment by jaidetree** [comment]
Cool. Cheers! :hand: (collaboration-five)

**Pull request merged**